test/integration/organizations_test.rb

```diff
@@ -0,0 +1,62 @@
+require "test_helper"
+
+class OrganizationsTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = create(:user, remember_token_expires_at: Gemcutter::REMEMBER_FOR.from_now)
+    post session_path(session: { who: @user.handle, password: PasswordHelpers::SECURE_TEST_PASSWORD })
+  end
+
+  test "should show an organization" do
+    organization = create(:organization, owners: [@user], handle: "arrakis", name: "Arrakis")
+    organization.rubygems << create(:rubygem, name: "arrakis", number: "1.0.0")
+
+    get "/organizations/#{organization.to_param}"
+
+    assert_response :success
+    assert page.has_content? "arrakis"
```

Comment:
These assertions that are checking the Organization name/handle is a bit unclear in terms of what's specifically looking for.

```suggestion
    assert_selector "h1", @organization.title
```

Reply:
Can't make this change in this type of test :( Looking forward to your refactor.